src/stringClass.js

```diff
@@ -0,0 +1,130 @@
+const { upperAndLowerConverter } = require('../helper/helper');
+
+String.prototype.hasVowels = function() {
+  var vowelsPattern = /[aeiou]/gi; // regular expression to find any vowel in a string
+
+  return vowelsPattern.test(this); // returns true if found else false
+};
+
+String.prototype.toUpper = function() {
+  let lowerCharRegex = /[a-z]/g; // regular expression pattern that find all lowerCase in a string
```

Comment:
I strongly advise against using var and let in one codebase. It comes off as a sign of inconsistency